airbyte-ci/connectors/base_images/poetry.lock

```diff
@@ -1712,24 +1712,15 @@ files = [
     {file = "PyYAML-6.0.1-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:69b023b2b4daa7548bcfbd4aa3da05b3a74b772db9e23b982788168117739938"},
     {file = "PyYAML-6.0.1-cp310-cp310-manylinux_2_17_s390x.manylinux2014_s390x.whl", hash = "sha256:81e0b275a9ecc9c0c0c07b4b90ba548307583c125f54d5b6946cfee6360c733d"},
     {file = "PyYAML-6.0.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:ba336e390cd8e4d1739f42dfe9bb83a3cc2e80f567d8805e11b46f4a943f5515"},
-    {file = "PyYAML-6.0.1-cp310-cp310-musllinux_1_1_x86_64.whl", hash = "sha256:326c013efe8048858a6d312ddd31d56e468118ad4cdeda36c719bf5bb6192290"},
```

Comment:
Doesn't look like it - looks like it appears if I `poetry lock --no-update` on master. Similar to the cryptography change. 

We should probably have a CI check that yells if `poetry lock --no-update` modifies files or something, because it definitely seems sus. I think in these deleted ones it's likely the case that poetry doesn't care (compare to if you change a version in the toml file and don't lock it, you get a `pyproject.toml changed significantly since poetry.lock was last generated. Run `poetry lock [--no-update]` to fix the lock file.` when you try to install)

Reply:
actually since that added cryptography back in im surprised it doesn't yell 🤔